board.py

```diff
@@ -6,7 +11,41 @@ class Stone(Enum):
     EMPTY = 0
 
 class Board:
+    """
+    This is a class to hold our game board object.
+    It doesn't matter how it is implemented as long as 
+    calling board.put_stone_at(1,1, Stone.BLACK) will later mean that
+    board.get_stone_at(1,1) == Stone.BLACK
+    """
+    def __init__(self, width, height):
+        """
+        This is called an initialiser. If you need to do some setup when the 
+        board is created with board = Board(), do it here.
+        perhaps making something to store the stones in might be worthwhile?
+        """
+        self.boardStone = [[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY],[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY],[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY],[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY],[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY]]
+        pass
     def get_stone_at(self, x, y):
-        return Stone.EMPTY
-    def put_stone_at(x, y, stone):
+        x=x-1
+        y=y-1
+        return self.boardStone[x][y]
+    def set_stone_at(self, x, y, stone):
+        x=x-1
+        y=y-1
+        if stone == Stone.BLACK and self.boardStone[x][y]==stone.EMPTY:
```

Comment:
`Stone` is the class, and `stone` is an instance. I think this will technically work as the class attributes are available from the instance, but it is confusing.